RocketChat/Rocket.Chat
# fix(i18n): add password placeholders for all .i18n.json files

## Proposed changes 

  This PR adds password placeholders in all `.i18n.json` files, ensuring consistency across the app's password input fields.
  The placeholder will be shown as eight dots (`********`), which helps improve user experience on login pages.


## Issue(s)
Closes #34032

## Steps to test or reproduce
1. Navigate to the `login` page.
2. Verify that the password field now shows a placeholder with eight dots (`********`).
3. Check that this change is reflected across all languages in the `.i18n.json` files.

## Further comments
This is my first contribution to Rocket.Chat. I would appreciate any guidance on setting up the local environment to test these changes effectively.

## Checklist
- [x] I have read the [Contributing Guide](https://github.com/RocketChat/Rocket.Chat/blob/develop/.github/CONTRIBUTING.md#contributing-to-rocketchat)
